app/assets/stylesheets/_state-file.scss

```diff
@@ -761,6 +761,10 @@ $state-colors: (
     .slab--footer {
       background-color: $color-nj-blue-darker;
     }
+
+    .review-header #household-info div:last-of-type {
```

Comment:
[DUST] Can you elaborate on what this hides and why this is necessary?

Reply:
This hides the edit button for name and dob, per the most recent comments on the ticket

Reply:
[dust] In the future, could you reach out when you are changing a shared component could you flag it for our attention? This change is acceptable since we will also be removing this button, but in the future I'd like to make sure we're on the same page.

In addition, I would prefer that the button be removed from the template entirely rather than being hidden so that it doesn't collect dust as a hidden element.

Reply:
This only hides the button for nj